.github/workflows/container.yml

```diff
@@ -1,9 +1,9 @@
-name: DCR Build Container Image
-
 on:
-  push:
-    paths-ignore:
-      - "apps-rendering/**"
+  workflow_call:
+    outputs:
+      container-image:
+        description: 'The generated container image path'
+        value: ${{ jobs.image.outputs.container-image }}
```

Comment:
I wasn't sure which step creates this output? Is it coming from "Push Image To GHCR"? ([L44](https://github.com/guardian/dotcom-rendering/pull/7503/files#diff-87249919b62f3d7f70416176a4254d5df6e59eab47c6de2a2a4fbbcf4138cbacR44))

Reply:
Comes from [L16](https://github.com/guardian/dotcom-rendering/pull/7503/files#diff-87249919b62f3d7f70416176a4254d5df6e59eab47c6de2a2a4fbbcf4138cbacL16) which comes from [L44](https://github.com/guardian/dotcom-rendering/pull/7503/files#diff-87249919b62f3d7f70416176a4254d5df6e59eab47c6de2a2a4fbbcf4138cbacR44)

Confusing because that output was already configured in a previous PR and so wasn't highlighted in this one!